x-pack/plugins/transform/public/app/sections/create_transform/components/aggregation_list/popover_form.tsx

```diff
@@ -265,6 +266,24 @@ export const PopoverForm: React.FC<Props> = ({ defaultData, otherAggNames, onCha
     formValid = validAggName && aggConfigDef.isValid();
   }
 
+  const isJSONValid = useMemo(() => {
+    if (
+      isFilterBooleanAgg(aggConfigDef) &&
```

Comment:
when you **_clone_** the transform with boolean filter, this check returns false and the page crashes